packages/kit/test/apps/basics/src/routes/alias/_tests.js

```diff
@@ -0,0 +1,8 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test) {
+	test('handles resolve.alias as array', '/alias', async ({ page }) => {
```

Comment:
could we unit test the config merging rather than having an integration test? the browser tests are kind of flaky and this seems like something we should be able to test without a server and browser

Reply:
yeah that makes sense after the change I made. Will move it as a unit test.

Reply:
Added a new unit test